SPDX-FileCopyrightText: 2022 fosslinux <fosslinux@aussies.space>

SPDX-License-Identifier: GPL-2.0-only

Remove hostname from libtool script.

--- libtool.m4	2022-02-26 10:25:54.265298039 +1100
+++ libtool.m4	2022-02-26 10:26:26.909344391 +1100
@@ -2357,8 +2357,6 @@
 
 # ### BEGIN LIBTOOL CONFIG
 
-# Libtool was configured on host `(hostname || uname -n) 2>/dev/null | sed 1q`:
-
 # Shell to use when invoking shell scripts.
 SHELL=$lt_SHELL
 
